pkg/tracer/tracer_linux.go

```diff
@@ -579,12 +579,13 @@ func (t *LinuxTracer) getMap(mapName string) (*bpflib.Map, error) {
 func (t *LinuxTracer) dispatchPerfEvent(event *common.PerfEvent) {
 	t.tcpConnInsightsLock.Lock()
 	defer t.tcpConnInsightsLock.Unlock()
-
+	connection := *event.Connection
 	if event.HTTPResponse != nil {
-		logger.Tracef("http response: %v", event.HTTPResponse)
+		logger.Warnf("http response: %v", event.Connection)
```

Comment:
trace

Reply:
This was for debugging, it has been reverted to trace in the latest commit